atest/robot/core/filter_by_names.robot

```diff
@@ -157,6 +157,8 @@ Filter Using Suite Long Name
     Run Suites    --suite Subsuites.Sub1
     Should Contain Suites    ${SUITE}    Subsuites
     Should Contain Tests   ${SUITE}   SubSuite1 First
+    Run Suites    --suite suite_should.be.parsed  ${DATA DIR}/misc
```

Comment:
This is a bit nitpicky, but we try to be consistent and use four spaces between arguments (unless aligning arguments otherwise makes sense). Here there's two spaces before the last argument, and also the newly added suites have just two space indentation.